### PR TITLE
Tabs content

### PR DESCRIPTION
**Pull request by shkolin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Fixed issues  | #27

### CODE REVIEW

File: docs/tabs.md [hunk 1]
```diff
@@ -83,3 +90,5 @@ Method                        | Description
 `size(string $value)`         | Size of the tabs list.                                                       | `''`
 `alignment(string $value)`    | Alignment of the tabs list.                                                  | `''`
 `style(string $value)`        | Style of the tabs list.                                                      | `''`
+`renderTabsContent(bool $value)` | Whether to render the `tabs-content` container and its content. | `true`
```
Comment (samdark) [inline docs/tabs.md:93]: Is there a use-case for not rendering tabs content if it's specified?

Comment (shkolin) [reply docs/tabs.md:93]: Do you mean that? https://github.com/yiisoft/yii-bulma/blob/b90569981b668b31ab7ba5a2aa47eb4bf1700d9e/tests/TabsTest.php#L334

Comment (samdark) [reply docs/tabs.md:93]: Yes. Why do we want to specify content but not render it at the same time?

Comment (shkolin) [reply docs/tabs.md]: Ready.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks!

**Pull request merged**